examples/data-objects/codemirror/src/codeMirror.ts

```diff
@@ -117,8 +98,9 @@ export class SmdeFactory implements IFluidDataStoreFactory {
 				]),
 			),
 			existing,
+			async (dataStoreRuntime: IFluidDataStoreRuntime) =>
+				CodeMirrorComponent.load(runtime, context, existing),
```

Comment:
The object (a `CodeMirrorComponent`) that was previously used as a router to handle requests in the data store runtime, and returned itself, now exposed directly in the data store's entryPoint.